src/theme/BlogPostPage/styles.module.css

```
@@ -1,3 +1,11 @@
+.header,
+.markdown,
+.footer {
+  max-width: 665px;
```

@goodroot after merging #1813 i tried to keep the same width as i found in your PR. But now i'm thinking maybe it's a bit too narrow? wdyt?

```suggestion
  max-width: 700px;
```

---

Agree!